### ISSUE TITLE
Comments of type '(*' do not take effect  if followed by a "("

### ISSUE THREAD

**Issue by raokun** [Serhioromano/vscode-st#64]
Comments of type '(*' do not take effect  if followed by a "(",like this.
![Snipaste_2024-12-10_17-57-39](https://github.com/user-attachments/assets/6c542ab6-5422-4b2d-ae0b-eaf581a7b0b0)

**Comment by Serhioromano** [comment]
There should be a whitespace. The simplest way is to press Shift+Alt+F for formatting and it will insert whitespace there.

**Comment by raokun** [comment]
> There should be a whitespace. The simplest way is to press Shift+Alt+F for formatting, and it will insert whitespace there.
> 
> Anyway, thank you. I'll have a look into it to see what I can do.

Sry，wo try to to press Shift+Alt+F for formatting, but it not work,like this.

![Image](https://github.com/user-attachments/assets/8b4bd8cd-2746-4b3b-afb0-51237bc4d1f8)

**Comment by Serhioromano** [comment]
Looks like i cannot find this bug. If the same start from new string it does not appear. Only at the end of another string.

**Comment by katarzynagorska** [comment]
Hi, I think i found similar issue with strings containing "(" - not sure if related or something different. You can see example in the picture below:

<img width="371" height="274" alt="Image" src="https://github.com/user-attachments/assets/972cd902-5e64-4f65-a464-9d35ac9a3cc0" />

**Comment by Serhioromano** [comment]
Yes it is similar. I was looking so hard but I cannot find the case of this artifact. I'll try something else.

**Comment by xjh22222228** [comment]
There is a similar issue here, if the comment is  `//测试 (V) `, the comment does not take effect because it contains non English and is enclosed in parentheses

**Comment by Serhioromano** [comment]
As a solution right now I can only recommend not to use () in comments. I cannot find the reason. Looks like I cannot fix it for now.

**Comment by Serhioromano** [comment]
Fixed
